chainreg/chainregistry.go

```diff
@@ -888,8 +888,15 @@ var (
 
 		BitcoinTestnetGenesis: {
 			{
-				"test.nodes.lightning.directory",
-				"soa.nodes.lightning.directory",
+				"test.nodes.lightning.wiki",
```

Comment:
Looks like signet also works already? Can we replace the above as well, since it hasn't worked in forever (was originally my domain).